index.js

```diff
@@ -93,6 +94,22 @@ module.exports = {
 
     this.ui.writeDebugLine('Generating preview-head.html');
 
+    const environment = parsedConfig.meta.find(meta => meta.name.endsWith('config/environment'));
```

Comment:
Can we write a test case for this?

Reply:
Yep! Just wanted to make sure you were cool with the feature first 😅